schemas/metaschema/domains.schema

```diff
@@ -1,5 +1,5 @@
 {
-  Id: { type: 'bigint', parse: x => new api.common.Uint64(x) },
+  Id: { type: 'bigint', parse: x => new common.Uint64(x) },
```

Comment:
I think this should be reverted	as AFAIK we have decided to bundle any libs under `api` in schemas.
/cc @tshemsedinov @belochub @nechaido